### PR TITLE
fix: replace requesthub slack with discord

### LINKED ISSUE #987: Replace RequestHub Slack with Discord
# Problem

“RequestHub” Slack is mentioned and linked several times throughout the `requestNetwork` repository. The RequestHub Slack workspace is deprecated in favor of the Request Network Discord. 

# Definition of Done

Submit a Pull Request (PR) in which all mentions of "RequestHub" are replaced with an invite to Discord: https://request.network/discord

### PR DESCRIPTION
**Pull request by PijiDev**
Fixes #987

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 This repository contains all the Request Network developer tools written in TypeScript.
 
-Join the [Request Hub][requesthub-slack-url] to get in touch with us.
+Join the [Request Discord][requesthub-discord-url] to get in touch with us.
 
 [website-url]: https://request.network
 [whitepaper-url]: https://request.network/assets/pdf/request_whitepaper.pdf
-[requesthub-slack-url]: https://request-slack.herokuapp.com/
+[requesthub-discord-url]: https://request.network/discord/
```
Comment (MantisClone) [inline README.md:17]: Please change this to `request-discord-url` as well.

File: README.md [hunk 2]
```diff
@@ -10,11 +10,11 @@
 
 This repository contains all the Request Network developer tools written in TypeScript.
 
-Join the [Request Hub][requesthub-slack-url] to get in touch with us.
+Join the [Request Discord][requesthub-discord-url] to get in touch with us.
```
Comment (MantisClone) [inline README.md:13]: just a minor quibble, can you change this to `request-discord-url` please?

**Review by MantisClone — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PijiDev) [reply README.md]: Fixed [b455df7](https://github.com/RequestNetwork/requestNetwork/pull/1032/commits/b455df7fc1989a2e6fcd5d3f361c26f2e3d9f155)

File: README.md (hunk 2 above)
Comment (PijiDev) [reply README.md]: Fixed [b455df7](https://github.com/RequestNetwork/requestNetwork/pull/1032/commits/b455df7fc1989a2e6fcd5d3f361c26f2e3d9f155)

**Review by MantisClone — APPROVED** [review]

**Comment by MantisClone** [comment]
@PijiDev these changes look good to me but the prettier formatter is complaining so the CI build fails.  Can you please run `yarn install` to make sure the pre-commit checks are installed, and then add a new commit that just adds a space somewhere so that the pre-commit checks run and auto-fix whatever is wrong?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ We welcome contributions from anyone, and are grateful for even the smallest of
 
 ## Got a Question or Problem?
 
-For quick help, join the [Request Hub Slack](https://request-slack.herokuapp.com/) and ask your question there.
+For quick help, join the [Request Discord]( https://request.network/discord/) and ask your question there.
```
Comment (MantisClone) [inline CONTRIBUTING.md:7]: Please remove the space after the first parenthesis `( https://request.network/discord/)`. This is what prettier is complaining about.

File: packages/request-client.js/README.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ const requestNetwork = new RequestNetwork({ useMockStorage: true });
 
 ## Guide
 
-We are currently writing the full API reference and more detailed guides. This section will be updated. If you need help in the meantime, [join the Request Hub Slack](https://request-slack.herokuapp.com/) and come chat with us.
+We are currently writing the full API reference and more detailed guides. This section will be updated. If you need help in the meantime, [join the Request Discord]( https://request.network/discord/) and come chat with us.
```
Comment (MantisClone) [inline packages/request-client.js/README.md:74]: Please remove the space after the first parenthesis `( https://request.network/discord/)`. This is what prettier is complaining about.

**Review by MantisClone — CHANGES_REQUESTED** [review]

**Review by PijiDev — COMMENTED** [review]
Fix spacing

**Comment by MantisClone** [comment]
This can be merged despite the `payment-detection` tests failing. They fail because they rely on secret API keys that PRs from external forks cannot access.

**Review by MantisClone — APPROVED** [review]

**Review by yomarion — APPROVED** [review]

**Comment by MantisClone** [comment]
@PijiDev one of your commits is not signed. Here's how I recommend you fix it:

1. [Configure git to use your SSH signing key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key)
    ```
    $ git config --global gpg.format ssh
    $ git config --global user.signingkey ~/.ssh/id_ed25519.pub
    ```
2. [Add your SSH signing key to Github](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account#adding-a-new-ssh-key-to-your-account)
    * Make sure to say it's a signing key and not an authentication key.
    * Example: image
4. [Configure git to sign all commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)
    ```
    git config --global commit.gpgsign true
    ```
5. [Go back and sign your commit using rebase](https://superuser.com/a/1123928)
    ```
    git rebase --exec 'git commit --amend --no-edit -n -S' -i master-copy
    ```
    * Normally, I'd recommend rebasing from `master`, but since you've modified your `master` branch, you'll need to create a new branch equivalent to upstream `master`. I named this branch `master-copy` in the command above.

**Comment by MantisClone** [comment]
@PijiDev Please disregard my previous comment. We removed the requirement for signed commits, so you no longer need to go back and sign the commits in this PR.

**Pull request merged**